crates/rooch-framework/sources/ord.move

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+module rooch_framework::ord {
```

Comment:
We can put the Bitcoin contracts in a separate directory.

Reply:
We have the plan to introduce a new `bitcoin_framwork.`